boards/nxp/mimxrt1060_evk/mimxrt1060_evk_mimxrt1062_qspi_C.overlay

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2022, Whisper.ai
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* FLEXPWM not routed to LED on this EVK */
+&flexpwm2_pwm3 {
+	status = "disabled";
```

Comment:
maybe would be also good to do /delete-node/ on the pwm leds nodes if there are no pwm leds on the board

Reply:
Yeah. This is a direct copy of the rev B dts. I want to review some stuff on this with the team.

I also have a local change for docs to discuss. I found one example in the tree that used tabs (which I personally like). There is also a consideration to change the default rev to something folks can actually buy.